builtin/game/knockback.lua

```diff
@@ -0,0 +1,46 @@
+-- can be overriden by mods
+core.calculate_knockback = function(player, hitter, time_from_last_punch, tool_capabilities, dir, distance, damage)
```

Comment:
```lua
function core.calculate_knockback( --
```

Reply:
There are no style rules for this.

Reply:
https://dev.minetest.net/Lua_code_style_guidelines#Misc (point 5)
>- Write function definitions of the form `function foo()` instead of the lambda form `foo = function()`, except when inserting functions in tables inline, where only the second form will work.

Reply:
Alright. Thanks for pointing that out.

Reply:
Fixed.